commit e5626eb80026c4b63f8682cdeca1456303c65791
Author: Ashutosh Dixit <ashutosh.dixit@intel.com>
Date:   Tue Feb 6 11:27:31 2024 -0800

    drm/xe/xe_gt_idle: Drop redundant newline in name
    
    Newline in name is redunant and produces an unnecessary empty line during
    'cat name'. Newline is added during sysfs_emit. See '27a1a1e2e47d ("drm/xe:
    stringify the argument to avoid potential vulnerability")'.
    
    v2: Add Fixes tag (Riana)
    
    Fixes: 7b076d14f21a ("drm/xe/mtl: Add support to get C6 residency/status of MTL")
    Reviewed-by: Riana Tauro <riana.tauro@intel.com>
    Signed-off-by: Ashutosh Dixit <ashutosh.dixit@intel.com>

diff --git a/drivers/gpu/drm/xe/xe_gt_idle.c b/drivers/gpu/drm/xe/xe_gt_idle.c
index 9358f7336889..9fcae65b6469 100644
--- a/drivers/gpu/drm/xe/xe_gt_idle.c
+++ b/drivers/gpu/drm/xe/xe_gt_idle.c
@@ -145,10 +145,10 @@ void xe_gt_idle_sysfs_init(struct xe_gt_idle *gtidle)
 	}
 
 	if (xe_gt_is_media_type(gt)) {
-		sprintf(gtidle->name, "gt%d-mc\n", gt->info.id);
+		sprintf(gtidle->name, "gt%d-mc", gt->info.id);
 		gtidle->idle_residency = xe_guc_pc_mc6_residency;
 	} else {
-		sprintf(gtidle->name, "gt%d-rc\n", gt->info.id);
+		sprintf(gtidle->name, "gt%d-rc", gt->info.id);
 		gtidle->idle_residency = xe_guc_pc_rc6_residency;
 	}